environments/production/common/rds.tf

```diff
@@ -1,111 +1,3 @@
-# Backend Postgres
-module "postgres" {
-  source = "../../../modules/rds"
-
-  environment    = var.environment
-  name           = "TradeTariffPostgres${title(var.environment)}"
-  engine         = "postgres"
-  engine_version = "13.15"
-
-  deletion_protection = true
-
-  instance_type = "db.m5.4xlarge"
-  multi_az      = true
-
-  backup_window      = "22:00-23:00"
-  maintenance_window = "Fri:23:00-Sat:01:00"
-  private_subnet_ids = data.terraform_remote_state.base.outputs.private_subnet_ids
-
-  allocated_storage     = 20
-  max_allocated_storage = 40
-  security_group_ids    = [module.alb-security-group.be_to_rds_security_group_id]
-
-  secret_kms_key_arn = aws_kms_key.secretsmanager_kms_key.arn
-
-  depends_on = [
-    module.alb-security-group
-  ]
-
-  tags = {
-    Name       = "TradeTariffPostgres${title(var.environment)}"
-    "RDS_Type" = "Instance"
-  }
-}
-
-
-module "read_only_postgres_connection_string" {
-  source          = "../../../modules/secret/"
-  name            = "postgres-read-only"
-  kms_key_arn     = aws_kms_key.secretsmanager_kms_key.arn
-  recovery_window = 7
-  secret_string   = "postgres://tariff_read:tariff@${module.postgres.userless_connection_string}"
-}
-
-# Admin Postgres
-module "postgres_admin" {
-  source = "../../../modules/rds"
-
-  environment    = var.environment
-  name           = "PostgresAdmin"
-  engine         = "postgres"
-  engine_version = "13.15"
-
-  deletion_protection = true
-
-  instance_type      = "db.t3.micro"
-  backup_window      = "22:00-23:00"
-  maintenance_window = "Fri:23:00-Sat:01:00"
-  private_subnet_ids = data.terraform_remote_state.base.outputs.private_subnet_ids
-
-  allocated_storage     = 20
-  max_allocated_storage = 40
-  security_group_ids    = [module.alb-security-group.be_to_rds_security_group_id]
-
-  secret_kms_key_arn = aws_kms_key.secretsmanager_kms_key.arn
-
-  depends_on = [
-    module.alb-security-group
-  ]
-
-  tags = {
-    Name       = "PostgresAdmin"
-    "RDS_Type" = "Instance"
-  }
-}
-
-module "postgres_commodi_tea" {
```

Comment:
So this is a separate database that we're running for the CommodiTea app, we'll have to migrate this to Aurora before removing it - doing this separately from the main RDS instance seemed the best way forward to minimise blast radius